src/canvas/SelectableCanvas.ts

```diff
@@ -772,17 +771,11 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
     obj: FabricObject,
     globalPointer: Point
   ): boolean {
-    if (
-      obj &&
-      obj.visible &&
-      obj.evented &&
-      // http://www.geog.ubc.ca/courses/klink/gis.notes/ncgia/u32.html
-      // http://idav.ucdavis.edu/~okreylos/TAship/Spring2000/PointInPolygon.html
-      obj.containsPoint(pointer)
-    ) {
+    if (obj && obj.visible && obj.evented && obj.containsPoint(pointer)) {
       if (
-        (this.perPixelTargetFind || obj.perPixelTargetFind) &&
-        !(obj as unknown as IText).isEditing
+        // if `obj` is selected we want it to be easy to interact with it
```

Comment:
i can see both ways being reasonable.
this will be a boolean option maybe called `preferBoundingBoxWhenActive: boolean` to add.

Reply:
IMO options less booleans more deleagtes/managers